src/js/apps/forms/form/form_app.js

```diff
@@ -249,14 +271,26 @@ export default App.extend({
     this.showChildView('formAction', new ReadOnlyView());
   },
   showFormSaveDisabled() {
-    this.showChildView('formAction', new SaveView({ isDisabled: true }));
+    this.showChildView('formAction', new SaveView({
+      isDisabled: true,
+      model: this.getState(),
+    }));
   },
   showFormSave() {
     if (this.isReadOnly) return;
-    const saveView = this.showChildView('formAction', new SaveView());
 
-    this.listenTo(saveView, 'click', () => {
-      Radio.request(`form${ this.form.id }`, 'send', 'form:submit');
+    const saveView = this.showChildView('formAction', new SaveView({
+      model: this.getState(),
+    }));
+
+    this.listenTo(saveView, {
+      'click:save'() {
+        Radio.request(`form${ this.form.id }`, 'send', 'form:submit');
+        this.showFormSaveDisabled();
```

Comment:
This ensures that both buttons are disabled when the user clicks the `Save` or `Save Form + Go Back` button.

<img width="211" alt="Screen Shot 2022-09-26 at 4 55 06 PM" src="https://user-images.githubusercontent.com/35355575/192387777-f80ff0eb-9214-47ac-bdc3-ac6410e6f25e.png">